src/mqueryfront/src/status/DatabaseTopology.js

```diff
@@ -71,6 +72,19 @@ class DatabaseTopology extends Component {
             });
     }
 
+    getDatasetsCompactingWarning(datasets) {
+        var fileCountThreshold = 1000 * 1000;
+        var amountThreshhold = 20;
+        var smallDatasets = datasets.filter(
+            (dataset) => dataset.file_count < fileCountThreshold
+        );
+        if (smallDatasets.length < amountThreshhold) {
+            return null;
+        }
+        return `At least ${amountThreshhold} datasets have less \
+        than ${fileCountThreshold} samples - consider compacting them.`;
```

Comment:
good idea about including the exact check in the warning message :slightly_smiling_face: